drivers/sensor/aht20/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2016 Intel Corporation
```

Comment:
Update to your own name and email here (or company) and year

Reply:
Fixed in Fixed in 82c64bf0dab4b88ef507d814bc70b795e95262fe.